[Bug] XGBoost could not be loaded

### To reproduce

In console:
```console
pytest pdr_backend/predictoor/
```

Returns the error message:
```text
XGBoost Library (libxgboost.dylib) could not be loaded.
```

### Details of setup

- MacOS running MacBook Pro M3

### How to fix - general

In console:
```console
brew install libomp
```

### TODO

- [ ] Add the "How to fix" instructions to MacOS part in predictoor.md and trader.md.

### Full traceback
```text
(venv) trentmc@trents-m3: ~/code/pdr-backend $ pytest pdr_backend/predictoor/
====================================================================================================== test session starts ======================================================================================================
platform darwin -- Python 3.12.3, pytest-8.2.2, pluggy-1.5.0
rootdir: /Users/trentmc/code/pdr-backend
configfile: pytest.ini
plugins: time-machine-2.14.2, env-1.1.3, dash-2.17.1, asyncio-0.21.1, typeguard-4.3.0, web3-6.20.0, requests-mock-1.12.1
asyncio: mode=Mode.STRICT
collected 9 items / 1 error                                                                                                                                                                                                     

============================================================================================================ ERRORS =============================================================================================================
_____________________________________________________________________________ ERROR collecting pdr_backend/predictoor/test/test_predictoor_agent.py _____________________________________________________________________________
pdr_backend/predictoor/test/test_predictoor_agent.py:19: in <module>
    from pdr_backend.predictoor.predictoor_agent import PredictoorAgent
pdr_backend/predictoor/predictoor_agent.py:15: in <module>
    from pdr_backend.aimodel.aimodel_factory import AimodelFactory
pdr_backend/aimodel/aimodel_factory.py:28: in <module>
    from xgboost import XGBClassifier, XGBRegressor
venv/lib/python3.12/site-packages/xgboost/__init__.py:6: in <module>
    from . import tracker  # noqa
venv/lib/python3.12/site-packages/xgboost/tracker.py:9: in <module>
    from .core import _LIB, _check_call, make_jcargs
venv/lib/python3.12/site-packages/xgboost/core.py:269: in <module>
    _LIB = _load_lib()
venv/lib/python3.12/site-packages/xgboost/core.py:222: in _load_lib
    raise XGBoostError(
E   xgboost.core.XGBoostError: 
E   XGBoost Library (libxgboost.dylib) could not be loaded.
E   Likely causes:
E     * OpenMP runtime is not installed
E       - vcomp140.dll or libgomp-1.dll for Windows
E       - libomp.dylib for Mac OSX
E       - libgomp.so for Linux and other UNIX-like OSes
E       Mac OSX users: Run `brew install libomp` to install OpenMP runtime.
E   
E     * You are running 32-bit Python on a 64-bit OS
E   
E   Error message(s): ["dlopen(/Users/trentmc/code/pdr-backend/venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.dylib, 0x0006): Library not loaded: @rpath/libomp.dylib\n  Referenced from: <6749C431-FD87-3912-A7CF-11CB9194D072> /Users/trentmc/code/pdr-backend/venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.dylib\n  Reason: tried: '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/opt/homebrew/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/libomp.dylib' (no such file), '/opt/homebrew/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/libomp.dylib' (no such file)"]
==================================================================================================== short test summary info ====================================================================================================
ERROR pdr_backend/predictoor/test/test_predictoor_agent.py - xgboost.core.XGBoostError: 
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================================================= 1 error in 7.82s ========================================================================================================
```

--- a/READMEs/predictoor.md
+++ b/READMEs/predictoor.md
@@ -54,7 +54,11 @@ curl https://raw.githubusercontent.com/oceanprotocol/contracts/main/addresses/ad
 If you're running MacOS, then in console:
 
 ```console
+# so that sapphire.py works. Details in #66
 codesign --force --deep --sign - venv/sapphirepy_bin/sapphirewrapper-arm64.dylib
+
+# so that xgboost works. Details in #1339
+brew install libomp
 ```
 
 ## 2. Simulate Modeling and Trading

--- a/READMEs/trader.md
+++ b/READMEs/trader.md
@@ -46,7 +46,11 @@ curl https://github.com/oceanprotocol/contracts/blob/main/addresses/address.json
 If you're running MacOS, then in console:
 
 ```console
+# so that sapphire.py works. Details in #66
 codesign --force --deep --sign - venv/sapphirepy_bin/sapphirewrapper-arm64.dylib
+
+# so that xgboost works. Details in #1339
+brew install libomp
 ```
 
 ## Simulate Modeling and Trading

